### PR TITLE
Implement export process + documentations

### PR DESCRIPTION
**Pull request by imranMnts**

### CODE REVIEW

File: src/Components/Details.tsx [hunk 1]
```diff
@@ -29,128 +39,357 @@ const excludedAttributes: Array<string> = [
...
+        />
+        <Text style={[{ color: theme.primaryColor, marginLeft: 6 }]}>Copy</Text>
+      </TouchableOpacity>
+    );
+  };
+
+  const _renderItems = (listOfItems: Array<[string, any]>) => {
+    return listOfItems
+      .filter((item: Array<string>) => !excludedAttributes.includes(item[0]))
+      .filter((item: Array<string>) => item[1] && item[1].length > 0)
```
Comment (imranMnts) [inline src/Components/Details.tsx:147]: we shoudl check item's length before `item[1]`

File: src/Components/Details.tsx [hunk 2]
```diff
@@ -29,128 +39,357 @@ const excludedAttributes: Array<string> = [
...
+      .map((item: Array<string>, index: number) => (
+        <View style={{ paddingHorizontal: 16 }} key={index}>
+          <View
+            style={{
+              marginBottom: 10,
```
Comment (imranMnts) [inline src/Components/Details.tsx]: should create a style in the StyleSheet

File: src/Components/Details.tsx [hunk 3]
```diff
@@ -29,128 +39,357 @@ const excludedAttributes: Array<string> = [
...
+              flexDirection: 'row',
+              alignItems: 'center',
+              justifyContent: 'space-between',
+            }}
+          >
+            <Text style={[{ color: theme.textColorFour }]}>{item[0]}</Text>
+            {item[0] === 'url' && _copyClipbutton(_copyToClipboard, item[1], 'URL has been copied to clipboard')}
```
Comment (imranMnts) [inline src/Components/Details.tsx:152]: why we check `item[0]` and we copy `item[1]`?

File: src/Components/Main.tsx [hunk 4]
```diff
@@ -86,145 +69,194 @@ export const Main = (props: IProps) => {
...
+          if (filter === request.method || filter === EnumFilterType.All) {
+            return (
+              request.type === EnumSourceType.ReactNativeRequest &&
+              (request.url?.toLowerCase().includes(searchQuery.toLowerCase()) ||
+                request.status === parseInt(searchQuery, 10) ||
+                request.method.toLowerCase() === searchQuery.toLowerCase())
+            );
+          }
+        } else {
+          if (filter === request.method || filter === EnumFilterType.All) {
```
Comment (imranMnts) [inline src/Components/Main.tsx]: what is the difference with `EnumSourceType.ReactNativeRequest` case to split it? Look like the same code

Comment (damien5701) [reply src/Components/Main.tsx]: If we choose to display only the NativeRequest, search must continue to work. That's why I have a duplicate here. However, after I'll do tests unit on this part, I'll try to do a refactor on this part.

File: src/Components/Details.tsx (hunk 3 above)
Comment (damien5701) [reply src/Components/Details.tsx:152]: That's normal, we test if the key is url. But, we want in the clipboard the value of the url (eg. http://myurl.com)

File: src/Components/Main.tsx (hunk 4 above)
Comment (damien5701) [reply src/Components/Main.tsx]: Done

File: src/Components/Details.tsx (hunk 1 above)
Comment (damien5701) [reply src/Components/Details.tsx:147]: Done

**Pull request merged**